src/index.html

```diff
@@ -5,9 +5,58 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/style.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" 
+    crossorigin="true">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <script type="module" src="https://unpkg.com/ionicons@7.1.0/dist/ionicons/ionicons.esm.js"></script>
+    <script nomodule="true" src="https://unpkg.com/ionicons@7.1.0/dist/ionicons/ionicons.js"></script>
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main class="main">
+      <div  class="card" data-qa="card">
+        <div class="card__top"> 
+          <img 
+          class="card__img" 
+            src="./images/imac.jpeg" 
+            alt="iMac"
+          >
+        </div>
+                    
+        <div class="card__bottom">
+          <div class="card__title">
+            <h2>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+            <h3>Product code: 195434</h3>
+          </div>
+        
+          <div class="card__review-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+        
+            <span class="card__review-count">Reviews: 5</span>
+          </div>
+        
+          <div class="card__price-container">
+            <span class="card__price-label">Price:</span>
+            <div class="card__price">$2,199</div>
```

Comment:
use semantic tag instead `p` or `span`
```suggestion
            <span class="card__price">$2,199</span>
```